first draft Services (#52)
DaanVanVugt/ignitioncomputing: _services/modelling.md

Author: @DaanVanVugt zou je hiervoor een aanzet kunnen geven?

---
layout: featured
title: Physics-Based Modelling
icon: /assets/img/icons/code-working-outline.svg
image: /assets/img/service/pexels-thisisengineering-3862132-min.jpg
description: We deliver physics based modelling applications
---

<div class="row">
    <div class="col-md-12">
        <div class="service-details mb-40">
            <h3>We build software for the future of science.</h3>
            <p>

>             Potentiam subtilitatis nostris lorem ipsum physicae exemplaribus liberamus, vestras ideas in rem convertentes. Innovate, simulare, et excellere cum nostro programmate provecto. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Possibilitates infinitas cum subtilitate et precisione. Futurem catenans cum technologia praeveniente reactorum fusionis nuclearis!

Reviewer: lipsum

---
layout: featured
title: Integrated Models and Software Interfacing
icon: /assets/img/icons/code-working-outline.svg
image: /assets/img/service/pexels-thisisengineering-3862132-min.jpg
description: We deliver integrated multiphysics modelling applications
---

<div class="row">
    <div class="col-md-12">
        <div class="service-details mb-40">
            <h3>Complex systems, complex data</h3>

>             <p> Nuclear fusion is a wide field, with many physics phenomena, many different experiments, calculations and simulations done by a diverse set of research groups. Data is generated over many geometries and physical parameters. As such, structured frameworks and interfaces are needed to ensure interoperability between different teams and data structures.</p>

Author: ```suggestion
            <p> Nuclear fusion is a vast field, with numerous physics phenomena, diverse experimental setups, complex calculations, and simulations conducted by various research teams.  Given that data is generated over a variety of geometries and physical parameters, structured frameworks and interfaces are necessary to ensure seamless interoperability between teams and their respective data structures.</p>
```

            <p>ITER is developing a standard for fusion data, the Integrated Modelling and Analysis Suite (IMAS).

>             We have developed a <a href="_posts/news/2021-03-21-ITER-IMAS-Python.md">pythonic, flexible interface</a> to data stored with the IMAS Access Layer, and are working on enabling access with the modern data processing ecosystem, using tooling such as xarray, netCDF, intake et cetera.

Author: ```suggestion
            We have developed a <a href="/news/2021/03/21/ITER-IMAS-Python">pythonic, flexible interface</a> for data stored with the IMAS Access Layer. We are now working on enabling access with the modern data processing ecosystem, using tooling such as xarray, netCDF, intake et cetera.
```

Author: De eerste zin loopt niet helemaal lekker, wat wordt er precies bedoelt? Dat we de acces layer ontwikkeld hebben? Of dat we een interface hebben gemaakt voor data dat met de IMAS access layer opgeslagen is?

Reviewer 2: Ik zou et cetera eruit laten

Reviewer 2: ```suggestion
            We have developed a <a href="/news/2021/03/21/ITER-IMAS-Python">pythonic, flexible interface</a> for data stored with the IMAS Access Layer. We are now working on enabling access with the modern data processing ecosystem, using tooling such as xarray, netCDF and intake.
```

            </p>
        </div>
    </div>
</div>
<div class="row">
    <div class="col-md-12">
        <div class="service-details mb-40">
            <h3>Multi-physics and multi-scale modelling</h3>
            <p>
            We have developed a <a href="/news/2022/08/31/ITER-Persistent-Actor-Framework">new IMAS integrated modelling simulation framework</a> built on <a href="https://muscle3.readthedocs.io/en/latest/">MUSCLE3</a>, which is currently used in JINTRAC and in the European Transport Simulator (ETS-PAF).
            </p>
            <p>

>             This framework is the base of the new ITER Pulse Design Simulator Tool that we are developing, in consortium with CEA and Université Côte d'Azur, which will combine magnetic equilibrium evolution simulation with core transport modelling, and integrate it with the ITER plasma control systems through synthetic diagnostics.

Author: ```suggestion
            This framework is the foundation of the ITER Pulse Design Simulator Tool, which we are developing in consortium with CEA and Université Côte d'Azur. This new simulation tool will combine magnetic equilibrium evolution simulation with core transport modelling, and integrate it with the ITER plasma control systems through synthetic diagnostics.
```

Als je fancy will done kan je ook 'novel simulation tool'  zeggen ipv 'new simulation tool'

            </p>
        </div>
    </div>
</div>
<div class="row">
    <div class="col-md-12">
        <div class="service-details mb-40">
            <h3>Digital twins</h3>
            <p>
            Building a predict-capable, understandable digital twin requires physics models, and may be aided by machine-learning methods but should not build solely upon them.

>             Complementing simulators with ML surrogates is a way to profit from potential performance increases while remaining certain of results.

Author: ChatGPT raad het volgende aan, maar ik ben nieet zeker of dat de originele betekenis denkt:

> Developing a predictive and interpretable digital twin requires robust physics-based models. While machine learning methods can aid this process, they should not serve as the sole foundation. Enhancing simulators with ML surrogates allows us to leverage potential performance gains while ensuring the reliability and accuracy of results

Reviewer 2: ```suggestion
            Developing an easy to interpret digital twin with predictive power, requires robust physics-based models. While machine learning methods can aid this process, they should not serve as the sole foundation. Enhancing simulators with ML surrogates allows us to leverage potential performance gains while ensuring the reliability and accuracy of results
```
